components/symbl_ai/actions/get-analytics/get-analytics.mjs

```diff
@@ -0,0 +1,39 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-get-analytics",
+  name: "Get Analytics",
+  description: "Get a list of metrics and member analytics (speaker ratio, talk time, silence, pace and overlap) from the conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/analytics/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+  },
+  async run({ $ }) {
+    try {
+      const {
+        metrics,
+        members,
+      } = await this.symblAIApp.getAnalytics({
+        $,
+        conversationId: this.conversationId,
+      });
+      $.export("$summary", `Successfully retrieved ${metrics.length} metrics and ${members.length} member${members.length === 1
+        ? ""
+        : "s'"} Analytics from the conversation`);
+      return {
+        metrics,
+        members,
+      };
+    } catch (error) {
+      console.log("Error: ", error);
+      $.export("$summary", "Failed to retrieve Analytics from the conversation");
+    }
```

Comment:
I think in [`symbl_ai.app.mjs`](https://github.com/PipedreamHQ/pipedream/blob/e562a06e81383a2f503a21999368fb0e87675214/components/symbl_ai/symbl_ai.app.mjs) there could be a method that throws a formatted error, such as the example below

```javascript
async makeRequest(customConfig) {
  ...

  try {
    return await axios($ || this, config);
  } catch (error) {
    this.throwFormattedError(error);
  }
},
throwFormattedError(error) {
  error = error.response;
  throw new Error(`${error.status} - ${error.statusText} - ${error.data.message}`);
},
```

Then the try/catch would no longer be necessary here

See these comments:

> I think the [`$summary`](https://pipedream.com/docs/components/guidelines/#guidelines-patterns) should use plain language to summarize what happened. E.g., "Failed to send the message. See the `unsent` [export](https://pipedream.com/docs/components/api/#returning-data-from-steps) for the unsent message."?
> Typically a `$summary` isn't emitted when an action fails. 
See https://github.com/PipedreamHQ/pipedream/pull/2394#discussion_r824025915

> Re-throw the caught `err` (or a [formatted error](https://github.com/PipedreamHQ/pipedream/blob/55236b3aa993cbcb545e245803d8654c6358b0a2/components/airtable/airtable.app.js#L71)) so the workflow job fails and the error gets sent to system notifications or the global error workflow?
See https://github.com/PipedreamHQ/pipedream/pull/2394#discussion_r824036802

Reply:
Awesome @andrewjschuang. I had the better error handling on my backlog but thanks for providing the guidance on this. It's addressed in 3ef55237.